.github/workflows/supported_storages_compatibility_check.yaml

```diff
@@ -0,0 +1,728 @@
+        with:
+          name: jdbc_mysql_5.7_integration_test_reports
+          path: core/build/reports/tests/integrationTestJdbc
+
+  jdbc-mysql-8-integration-test:
+    runs-on: ubuntu-latest
+
+    services:
+      mysql:
+        image: mysql:8.1
+        env:
+          MYSQL_ROOT_PASSWORD: mysql
+        ports:
+          - 3306:3306
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up JDK 8
+        uses: actions/setup-java@v3
+        with:
+          java-version: '8'
+          distribution: 'temurin'
+
+      - name: Setup and execute Gradle 'integrationTestJdbc' task
+        uses: gradle/gradle-build-action@v2
+        with:
+          arguments: integrationTestJdbc
+
+      - name: Upload Gradle test reports
+        if: always()
+        uses: actions/upload-artifact@v3
+        with:
+          name: jdbc_mysql_8_integration_test_reports
+          path: core/build/reports/tests/integrationTestJdbc
+
+  jdbc-postgresql-12-integration-test:
+    runs-on: ubuntu-latest
+
+    services:
+      mysql:
+        image: postgres:12.16-alpine
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+        ports:
+          - 5432:5432
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up JDK 8
+        uses: actions/setup-java@v3
+        with:
+          java-version: '8'
+          distribution: 'temurin'
+
+      - name: Setup and execute Gradle 'integrationTestJdbc' task
+        uses: gradle/gradle-build-action@v2
+        with:
+          arguments: integrationTestJdbc -Dscalardb.jdbc.url=jdbc:postgresql://localhost:5432/ -Dscalardb.jdbc.username=postgres -Dscalardb.jdbc.password=postgres
+
+      - name: Upload Gradle test reports
+        if: always()
+        uses: actions/upload-artifact@v3
+        with:
+          name: jdbc_postgresql_12_integration_test_reports
+          path: core/build/reports/tests/integrationTestJdbc
+
+  jdbc-postgresql-13-integration-test:
+    runs-on: ubuntu-latest
+
+    services:
+      mysql:
+        image: postgres:13.12-alpine
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+        ports:
+          - 5432:5432
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up JDK 8
+        uses: actions/setup-java@v3
+        with:
+          java-version: '8'
+          distribution: 'temurin'
+
+      - name: Setup and execute Gradle 'integrationTestJdbc' task
+        uses: gradle/gradle-build-action@v2
+        with:
+          arguments: integrationTestJdbc -Dscalardb.jdbc.url=jdbc:postgresql://localhost:5432/ -Dscalardb.jdbc.username=postgres -Dscalardb.jdbc.password=postgres
+
+      - name: Upload Gradle test reports
+        if: always()
+        uses: actions/upload-artifact@v3
+        with:
+          name: jdbc_postgresql_13_integration_test_reports
+          path: core/build/reports/tests/integrationTestJdbc
+
+  jdbc-postgresql-14-integration-test:
```

Comment:
It looks like PostgreSQL 15 is already released:
https://www.postgresql.org/support/versioning/

Can you please add this version to this CI?

Reply:
I added it.